### PR TITLE
FOUNDATIONS:  Retrieve All `Time`

### LINKED ISSUE #224: FOUNDATIONS: Retrieve All `Time`

### PR DESCRIPTION
**Pull request by Ulugbek4499**
Closes #224

### CODE REVIEW

File: Tarteeb.Api.Tests.Unit/Services/Foundations/TimeSlots/TimeServiceTests.Logic.RetrieveAll.cs [hunk 1]
```diff
@@ -0,0 +1,35 @@
+﻿using System;
```
Comment (ShohruhUzDev) [inline Tarteeb.Api.Tests.Unit/Services/Foundations/TimeSlots/TimeServiceTests.Logic.RetrieveAll.cs:1]: add header file

File: Tarteeb.Api/Services/Foundations/Times/TimeService.cs [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+
+using System.Linq;
+using Tarteeb.Api.Brokers.Storages;
+using Tarteeb.Api.Models.Foundations.Times;
+
+namespace Tarteeb.Api.Services.Foundations.Times
+{
+    public class TimeService:ITimeService
+    {
+        IStorageBroker storageBroker;
```
Comment (ShohruhUzDev) [inline Tarteeb.Api/Services/Foundations/Times/TimeService.cs:14]: add "private readonly"

**Comment by ShohruhUzDev** [comment]
Menimcha Testni Exceptions qismini ham yozishiz kerak

File: Tarteeb.Api.Tests.Unit/Services/Foundations/TimeSlots/TimeServiceTests.Logic.RetrieveAll.cs [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+        [Fact]
+        public void ShouldRetrieveAllTimes()
+        {
+            //given
+            IQueryable<Time> randomTimes = CreateRandomTimes();
+            IQueryable<Time> storageTimes = randomTimes;
+            IQueryable<Time> expectedTimes = storageTimes;
+
+            this.storageBrokerMock.Setup(broker=>
+                broker.SelectAllTimes()).Returns(storageTimes);
```
Comment (Javlonbek-dev) [inline Tarteeb.Api.Tests.Unit/Services/Foundations/TimeSlots/TimeServiceTests.Logic.RetrieveAll.cs:21]: add empty line

**Review by Javlonbek-dev — CHANGES_REQUESTED** [review]

**Pull request closed without merging**